db.rb

```diff
@@ -1,9 +1,49 @@
 require "active_record"
 
-ActiveRecord::Base.establish_connection(
-  adapter: "sqlite3",
-  database: "db/db.sqlite3"
-)
+def setup_db(db_location)
+  ActiveRecord::Base.establish_connection(
+    adapter: "sqlite3",
+    database: db_location
+  )
+end
+
+def wipe_db
+  User.delete_all
+  Comment.delete_all
+  Regex.delete_all
+  Reason.delete_all
+  Room.delete_all
+  WhitelistedUser.delete_all
+end
+
+def record_comment(comment, perspective_score:)
```

Comment:
Why has this moved over to db.rb? And if it's here anyways, why doesn't it live under the `Comment` model?